.travis.yml

```diff
@@ -1,2 +1,10 @@
 language: rust
 rust: nightly
+cache: cargo
+before_script:
+- export PATH="$PATH:$HOME/.cargo/bin"
+- which rustfmt || cargo install rustfmt
+script:
+- cargo fmt -- --write-mode=diff
```

Comment:
That's fine. The alternative would be to use a post-commit hook to re-format the code in-situ, but I think I prefer how you've done it here.